<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgScriptParameters</class>
 <widget class="QDialog" name="CQDlgScriptParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Parameters</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QToolButton" name="qqAddNew">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>10</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Add new parameter</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Script parameters</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QListWidget" name="qqParameterList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameter properties</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqValue">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>261</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Unit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqUnit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>261</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqPrivate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Parameter is private</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqPersistent">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Parameter is persistent</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="qqClose">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>280</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqAddNew</tabstop>
  <tabstop>qqParameterList</tabstop>
  <tabstop>qqValue</tabstop>
  <tabstop>qqUnit</tabstop>
  <tabstop>qqPrivate</tabstop>
  <tabstop>qqPersistent</tabstop>
  <tabstop>qqClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
